src/components/RestaurantDetails/components/Offers/Offers.js

```diff
@@ -0,0 +1,53 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import offerReader from './readers/offerReader';
+
+//Icons
+import offerIcon from '../../../../images/offer.svg';
+
+//CSS
+import './Offers.css';
+
+function renderOffers(offers) {
+  return offers.map((offer) => {
+    const discount = offerReader.discount(offer);
+    const coupon = offerReader.coupon(offer);
+    return (
```

Comment:
may be you can keep it as a separate component because there's a good amount of JSX